src/components/TextField/TextField.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Input, Err } from './style';
+
+export const TextField = (props) => {
+  const {
+    error, arg, dis,
+  } = props;
+
+  const outPut = error ? (
+    <>
+      <Input type="text" placeholder={arg} error />
+      <Err> Could not be greater than 100 </Err>
+    </>
+  ) : (
+    <>
+      <Input type="text" placeholder={arg} disabled={dis} />
+      <br />
+    </>
+  );
+
+  return outPut;
```

Comment:
we can omit the use of an extra variable "outPut" by this: 

return (error ? (
    <>
      <Input type="text" placeholder={arg} error />
      <Err> Could not be greater than 100 </Err>
    </>
  ) : (
    <>
      <Input type="text" placeholder={arg} disabled={dis} />
      <br />
    </>
  ))